examples/dodge-the-creeps/rust/src/player.rs

```diff
@@ -58,6 +60,14 @@ impl Area2DVirtual for Player {
     }
 
     fn process(&mut self, delta: f64) {
+        if Engine::singleton().is_editor_hint() {
+            // Don't process if running in editor
+            // This part should be removed when this
+            // issue is resolved: https://github.com/godot-rust/gdext/issues/70
+
+            return;
+        }
```

Comment:
Small request, could you put the comment on 2 lines and before the `if`?
Thanks 🙂